regression/cbmc/pointer-to-char-array-function-parameters/test.c

```diff
@@ -0,0 +1,119 @@
+#include <assert.h>
+#include <stddef.h>
+
+int is_printable_char(char c)
+{
+  switch(c)
+  {
+  case ' ':
```

Comment:
Could this be done instead with a few conditions? e.g `if(c >= 'a' && c <= 'z') { ... }` ?

Reply:
Yes